### ISSUE TITLE
Set target point manually

### ISSUE THREAD

**Issue by MFlisar** [deano2390/MaterialShowcaseView#1]
I've a full width `CheckedTextView` and when highlighting it I would like to highlight the checkbox for example. I can set the radius manually, that's already nice. But I would need to set the relative position of the circle to it's view as well.

I can't do that right now, can I?

Anything planned in this direction? Setting a gravity would maybe also fulfill most of the usecases, offering left, right, top, bottom, center would be enough for me. Although, setting the point manually is more flexible.

**Comment by deano2390** [comment]
Sounds like a reasonable idea, i'll have a think and see what I can come up with.

**Comment by MFlisar** [comment]
Great, thanks

**Comment by deano2390** [comment]
As a workaround you could wrap the CheckedTextView in a relative layout, place an invisible View in with the appropriate gravity and then target that view with the Showcase.

**Comment by MFlisar** [comment]
Good idea. I've many views like that and use them in a complex settings fragment and I anyway create all views from one view.xml. The workaround would be easy to integrate for me.
